modules/host/errors_test.go

```diff
@@ -0,0 +1,330 @@
+package host
+
+import (
+	"bufio"
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/modules"
+)
+
+// countFileLines is a helper function that will count the number of lines in a
+// file, based on the number of '\n' characters. countFileLines will load the
+// file into memory using ioutil.ReadAll.
+//
+// countFileLines will ignore all lines with the string 'DEBUG' in it.
+func countFileLines(filepath string) (uint64, error) {
```

Comment:
Alternatively we could just have the logger itself do all the probabilistic logging, it could look for a special error interface that has Type() and Probability() functions